plugins/GSdx/GSOsdManager.cpp

```diff
@@ -0,0 +1,445 @@
+/*
+ *	Copyright (C) 2014-2016 Gregory hainaut
+ *	Copyright (C) 2016-2016 Jason Brown
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with GNU Make; see the file COPYING.  If not, write to
+ *  the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA USA.
+ *  http://www.gnu.org/copyleft/gpl.html
+ *
+ */
+
+#include "stdafx.h"
+#include "GSdx.h"
+#include "GSOsdManager.h"
+
+void GSOsdManager::LoadFont() {
+	FT_Error error = FT_New_Face(m_library, theApp.GetConfigS("osd_fontname").c_str(), 0, &m_face);
+	if (error == FT_Err_Unknown_File_Format) {
+		fprintf(stderr, "Failed to init the freetype face: unknwon format\n");
+    m_face = NULL;
```

Comment:
File is indented with tabs, but this line is indented with spaces. There's a few more instances of this happening in this file.